common/Command.proto

```diff
@@ -74,6 +74,28 @@ import "common/Common.proto";
 //                 MaxResponseSize: Integer
 //         ------------------------
 //     FixedTriggerDuration: Long
+//
+// Name: ContinuousProfilingPolicyQuery
+// Args:
+//     ServiceWithPolicyJSON: List JSON serialization of ServiceWithPolicy.
+//         --- ServiceWithPolicy ---
+//         ServiceName: String
+//         UUID: String
+//         Profiling: Multiple profiling configuration. Map
+//             Key: Profiling type. Enum, value = ON_CPU, OFF_CPU, NETWORK
+//             Value: Profiling policies. Map
+//                 Key: Monitoring type. Enum, value = PROCESS_CPU, PROCESS_THREAD_COUNT, SYSTEM_LOAD, PROCESS_RESPONSE_ERROR_RATE, PROCESS_RESPONSE_AVG_TIME
+//                 Value: Policy configuration. Object.
+//                     Threshold: String
+//                     Period(s): Integer
+//                     Count: Integer
+//                     URIList: List<String>
+//                     URIRegexList: List<String>
```

Comment:
https://github.com/apache/skywalking-data-collect-protocol/pull/83/files#r1102879051

This reminds me, Regex should be an expression. We don't need a list, as the regex could represent a list already, right?